main.tf

```diff
@@ -56,6 +37,53 @@ EOF
   containers = concat([local.container], var.additional_containers)
 }
 
+resource "aws_ecr_lifecycle_policy" "ramp_ecr_lifecycle_policy" {
```

Comment:
Change the resource name from `ramp_ecr_lifecycle_policy` to `ecr_lifecycle_policy` only

Reply:
Done